views/index.erb

```diff
@@ -55,5 +193,71 @@
   // TODO - Count the number of words in the profile.
   // hint - look at what the innerHTML property provides.
 
+  var about = profile_values[0].innerHTML;
```

Comment:
Each of these sections feels incredibly similar. How could you reduce some of this duplication? Perhaps you could extract a function for the common elements and rely on this instead?